.storybook/main.js

```diff
@@ -8,10 +13,32 @@ module.exports = {
   ],
   // https://storybook.js.org/docs/react/configure/typescript#mainjs-configuration
   typescript: { check: false },
-  webpackFinal: async (config) => {
-    // Make our `src/...` imports work
-    config.resolve.modules.push(__dirname, "./");
-    return config;
+  async viteFinal(config) {
+    const mergedConfig = mergeConfig(config, {
+      esbuild: {
+        // ignoring console warns of "Top-level "this" will be replaced with undefined since this file is an ECMAScript module"
+        // See https://github.com/vitejs/vite/issues/8644#issuecomment-1159308803
+        logOverride: { "this-is-undefined-in-esm": "silent" },
+      },
+      resolve: {
+        alias: {
+          src: path.resolve(__dirname, "../src"),
+        },
+      },
+    });
+
+    return {
+      ...mergedConfig,
+      plugins: [
+        // Filter out `vite:react-jsx` per suggestion in `plugin-react`...
```

Comment:
Is this something we should remove from our main `vite.config.js`?

Reply:
There is no main `vite.config.js` in this repo. Are you talking about `internal-frontend`?